### PR TITLE
Add systemd script for Ubuntu 15.04

### PR DESCRIPTION
**Pull request by jcheng5**
The systemd script could probably be a lot simpler; we shouldn't
need a PIDFILE if we're not running in daemon mode. And I have a
feeling we shouldn't be writing into /var/log/shiny-server.log for
the main server, but rather to the systemd journal.

However, in the interest of getting a working build out to 15.04
users ASAP, and not having to retest Red Hat 7, we'll just take
our existing systemd file with minimal changes.

### CODE REVIEW

**Comment by jcheng5** [comment]
Tested on:

* [x] Ubuntu 15.04
* [ ] Ubuntu >12.04/<15.04
* [ ] Ubuntu 12.04
* [ ] CentOS 6
* [ ] CentOS 7

Todo:

* [ ] Documentation

**Comment by jcheng5** [comment]
@fereshtehRS @trestletech For Upstart (and init.d) we added `LANG=en_US.UTF-8` (or another value). Do we need the same for systemd? On 15.04 I get this result when printing `Sys.getlocale()` from a Shiny app:

```
LC_CTYPE=en_US.UTF-8;LC_NUMERIC=C;LC_TIME=en_US.UTF-8;LC_COLLATE=en_US.UTF-8;LC_MONETARY=en_US.UTF-8;LC_MESSAGES=en_US.UTF-8;LC_PAPER=en_US.UTF-8;LC_NAME=C;LC_ADDRESS=C;LC_TELEPHONE=C;LC_MEASUREMENT=en_US.UTF-8;LC_IDENTIFICATION=C

File: vagrant/ubuntu15.04/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  # All Vagrant configuration is done here. The most common configuration
+  # options are documented and commented below. For a complete reference,
+  # please see the online documentation at vagrantup.com.
+
+  # Every Vagrant virtual environment requires a box to build off of.
+  config.vm.box = "ubuntu/vivid64"
```
Comment (aronatkins) [inline vagrant/ubuntu15.04/Vagrantfile]: do you need to specify a url?

**Comment by aronatkins** [comment]
Ok by me, but leaving the LANG stuff for others to review.

**Comment by trestletech** [comment]
I don't remember the entire context, but I'm going from https://github.com/rstudio/shiny-server/pull/75 and https://github.com/rstudio/shiny-server/pull/73 and it seems to be something we were at least intentional about. Not sure what system configurations would result in not having a LANG or why that's more of a concern only on Ubuntu, but I feel like it wouldn't be a bad idea to add that LANG logic here, too.

File: vagrant/ubuntu15.04/Vagrantfile (hunk 1 above)
Comment (jcheng5) [inline vagrant/ubuntu15.04/Vagrantfile]: Apparently not. It seems to automatically get pulled from Atlas now, much like the docker hub or whatever.

**Comment by jcheng5** [comment]
Added LANG, and tested that it does indeed set LANG in the shiny-server process.

I realized that the docs are in the Pro repo, so I'll update the docs when we merge these changes over to Pro.

Should I go ahead and merge?

**Comment by trestletech** [comment]
:+1:

**Pull request merged**